docs/user-guide/src/bmo/introduction.md

```diff
@@ -1,23 +1,77 @@
 # Bare Metal Operator
 
-The Bare Metal Operator (BMO) is a custom Kubernetes controller that deploys baremetal hosts, represented in Kubernetes by BareMetalHost (BMH), as Kubernetes nodes. To this end Ironic is used.
+The Bare Metal Operator (BMO) is a Kubernetes controller that manages
+bare-metal hosts, represented in Kubernetes by `BareMetalHost` (BMH) *custom
```

Comment:
@dtantsur I thought you preferred it here

Reply:
or you didnt change it but this line still says BMH an acronym, so I preferred the same throughout the text. I am ok either way

Reply:
I put it here just as a synonym, but maybe you have a point: we should be consistent. What do you think it more readable?

Reply:
I am fine with `BareMetalHost` TBH